learn/cookbooks/update_cloud.md

```diff
@@ -0,0 +1,249 @@
+:::
+
+## Step 1: Verify your database version
+
+Before we begin, you need to verify the version of Meilisearch that's compatible with your database, in other words, the version that indexed the data.
+
+Use the get version endpoint to check your Meilisearch version:
+
+```sh
+curl \
+  -X GET 'http://<your-domain-name>/version' \
+  -H 'Authorization: Bearer API_KEY'
+```
+
+::: warning
+If Meilisearch returns a [`missing_authorization_header`](/reference/errors/error_codes.md#missing-authorization-header) error code, you might be using v0.24 or below. Change the authorization header to `X-MEILI-API-KEY: apiKey`:
+
+```sh
+curl \
+  -X GET 'http://<your-domain-name>/version' \
+  -H 'X-Meili-API-Key: API_KEY'  
+
+```
+
+:::
+
+The response should look something like this:
+
+```json
+{
+  "commitSha": "stringOfLettersAndNumbers",
+  "commitDate": "YYYY-MM-DDTimestamp",
+  "pkgVersion": "x.y.z"
+}
+```
+
+::: note
+If you are updating to v0.28 or above, keys imported from the old version will have their `key` and `uid` fields regenerated
+:::
+
+If your `pkgVersion` is 0.21 or above, you can jump to [step 3](#step-3-create-the-dump). If not, proceed to the next step.
+
+## Step 2: Set all fields as displayed attributes
+
+::: warning
+This step is only required if you are using v0.20 or below.
+:::
+
+When creating dumps using Meilisearch versions v0.20 or below, all fields must be [displayed](/learn/configuration/displayed_searchable_attributes.md) to be saved in the dump.
+
+Start by verifying that all attributes are included in the displayed attributes list:
+
+```sh
+# replace {index_uid} with your index's unique id
+curl \
+  -X GET 'http://<your-domain-name>/indexes/{index_uid}/settings/displayed-attributes' \
+  -H 'X-Meili-API-Key: API_KEY'  
+
+```
+
+If the response is `{'displayedAttributes': '["*"]'}`, you can proceed on to [step 3](#step-3-create-the-dump).
+
+If it's something else, save the current list of displayed attributes to restore it after the migration is complete. Next, reset the list of displayed attributes to its default value (`["*"]`) using:
+
+```sh
+curl \
+  -X DELETE 'http://<your-domain-name>/indexes/{index_uid}/settings/displayed-attributes' \
+  -H 'X-Meili-API-Key: API_KEY'  
+```
+
+This command returns an `updateId`. You can use this to track the status of the operation.
+
+```sh
+# replace {updateId} with the updateId returned by the previous request
+curl \
+  -X GET 'http://<your-domain-name>/indexes/{index_uid}/updates/{updateId}' \
+  -H 'X-Meili-API-Key: API_KEY'  
+```
+
+Once the status updates to `processed`, you're good to go.
+
+## Step 3: Create the dump
+
+Before creating your dump, ensure that your [dump directory](/learn/configuration/instance_options.md#dumps-directory) is accessible. By default, dumps are created in a folder called `dumps` in the configuration file directory: `/var/opt/meilisearch/dumps`
```

Comment:
>By default, dumps are created in a folder called `dumps` in the configuration file directory: `/var/opt/meilisearch/dumps`

Not sure if it's relevant to the user, but it's not in this folder by default it's been configured to be in this folder by default for the cloud provider

Reply:
>it's been configured to be in this folder by default for the cloud provider

Does this apply to all cloud providers, @alallema?

Reply:
So something like this would be correct?

>By default, cloud platforms are configured to store dumps in the `/var/opt/meilisearch/dumps` directory

Reply:
>Does this apply to all cloud providers, @alallema?

Yes, all cloud providers are configured the same way.

>By default, cloud platforms are configured to store dumps in the /var/opt/meilisearch/dumps directory

Perfect! That said I said it just for information, but I leave it to your appreciation if yes or no it is relevant to tell the user, maybe it's not ...